components/script/timers.rs

```diff
@@ -418,6 +418,12 @@ enum InternalTimerCallback {
     ),
 }
 
+impl Default for JsTimers {
```

Comment:
Please don't add dead code. I think you can replace all calls `JSTimers::new()` with `JSTimers::default()` everywhere and move the contents of `new()` into `default()`.

Reply:
Thank you @mrobinson 
Indeed the implementation of  ```JsTimers::default``` behavior is unused.
However won't the change and replacement of ```new()``` with ```default()``` have effects on the use of ```JsTimers::new()```

Reply:
Yes, every usage of `JSTimers::new()` should be replaced with `JSTimers::default()`.

Reply:
> Thank you @mrobinson Indeed the implementation of `JsTimers::default` behavior is unused. However won't the change and replacement of `new()` with `default()` have effects on the use of `JsTimers::new()`

Apologies @mrobinson 
I have reread your review and it does make sense.